test/k8s-integration/driver.go

```diff
@@ -65,11 +77,18 @@ func installDriver(goPath, pkgDir, stagingImage, stagingVersion, deployOverlayNa
 	if err != nil {
 		return fmt.Errorf("failed to deploy driver: %v", err)
 	}
-
+	klog.Infof("Deploying driver")
+	fmt.Printf("Deploying driver")
 	// TODO (#139): wait for driver to be running
-	time.Sleep(time.Minute)
-	statusCmd := exec.Command("kubectl", "describe", "pods", "-n", driverNamespace)
-	err = runCommand("Checking driver pods", statusCmd)
+	if platform == "windows" {
+		klog.Infof("Test waiting for 15 mins")
```

Comment:
Can this be move descriptive, like "Waiting for 15 minutes to start the driver"?

Also it really takes 15 minutes?

Reply:
I have seen it took about 12 mins before. Normally downloading images take longer time than linux ones. I will try to get more data about this.